tests/end-to-end/director/labeling_api_test.go

```diff
@@ -467,6 +467,60 @@ func TestDeleteLabelDefinition(t *testing.T) {
 		saveQueryInExamples(t, deleteLabelDefinitionRequest.Query(), "delete label definition")
 	})
 
+	t.Run("Delete Label Definition while it's being used by some labels with deleteRelatedLabels parameter set to true - should succeed", func(t *testing.T) {
+
+		t.Log("Create LabelDefinition")
+		createLabelDefinitionRequest := fixCreateLabelDefinitionRequest(ldInputGql)
+		ld := graphql.LabelDefinition{}
+
+		err = tc.RunQuery(ctx, createLabelDefinitionRequest, ld)
+		require.NoError(t, err)
+
+		t.Log("Create application")
+		app := createApplication(t, ctx, "app")
+		defer deleteApplication(t, app.ID)
+
+		t.Log("Create runtime")
```

Comment:
We could create Runtime and Application along with labels - it will reduce code here in the test

Reply:
now I use `setRuntimeLabel` and `setApplicationLabel` functions